src/main/java/io/cdap/plugin/salesforce/plugin/SalesforceConnectorConfig.java

```diff
@@ -78,19 +79,17 @@ public class BaseSalesforceConfig extends ReferencePluginConfig {
   @Description("Endpoint to authenticate to")
   @Macro
   @Nullable
-  private String loginUrl;
+  public String loginUrl;
```

Comment:
nit: No need to make it public, just use the getter - getLoginUrl()

Reply:
done